content/en/docs/setup/platform-setup/huaweicloud/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Huawei Cloud
+description: Instructions to setup an Huawei Cloud kubernetes cluster for Istio.
+weight: 23
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/huaweicloud/
+    - /docs/setup/kubernetes/platform-setup/huaweicloud/
+keywords: [platform-setup,huawei,huaweicloud,cce]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+Follow these instructions to prepare a cluster for Istio using the
+[Huawei Cloud Container Engine
+](https://www.huaweicloud.com/intl/product/cce.html).
+You can deploy a Kubernetes cluster to Huawei Cloud quickly and easily in the
+`Cloud Container Engine Console`, which fully supports Istio.
+
+{{< tip >}}
+Huawei offers a {{< gloss >}}managed control plane{{< /gloss >}} add-on for the Huawei Cloud Container Engine,
+which you can use instead of installing Istio manually.
+Refer to [Huawei Application Service Mesh](https://support.huaweicloud.com/istio/)
+for details and instructions.
+{{< /tip >}}
+
+Following the [Huawei Cloud Instructions](https://support.huaweicloud.com/en-us/qs-cce/cce_qs_0008.html) to prepare a cluster before manually installing Istio, proceed as follows:
```

Comment:
Should the user follow the instructions in the link? I'm guessing not, but it's a little confusing.

Reply:
This is an instruction about how to create a k8s using huawei cce